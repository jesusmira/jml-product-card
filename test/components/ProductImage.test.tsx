import React from 'react';
import renderer from 'react-test-renderer';
import { ProductCard,  ProductImage } from '../../src/components';
import { product2 } from './data/products';


describe('Pruebas en <ProductImage/>', () => { 

    test('debe de mostrar el componente inicialmente del producto', () => { 
        
        const warpper = renderer.create(
            <ProductImage img='https://test.jpg'/>
        )

        expect(warpper.toJSON()).toMatchSnapshot();


     });

    test('Debe de mostrar el componente con la imagen del producto 2', () => { 

        const warpper = renderer.create(
            <ProductCard product={ product2}>
                {
                    () => (
                        <ProductImage/>
                    )
                }
            </ProductCard>
        )

        expect(warpper.toJSON()).toMatchSnapshot();

     })

 })