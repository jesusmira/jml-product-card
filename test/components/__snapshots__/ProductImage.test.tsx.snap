// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <ProductImage/> Debe de mostrar el componente con la imagen del producto 2 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="Product Image"
    className="productImg undefined"
    src="./coffee-mug.png"
  />
</div>
`;

exports[`Pruebas en <ProductImage/> debe de mostrar el componente inicialmente del producto 1`] = `
<img
  alt="Product Image"
  className="productImg undefined"
  src="https://test.jpg"
/>
`;
